communication/contributor-comms/automation/announcement-guidelines.md

```diff
@@ -0,0 +1,49 @@
+# K8s Announcement Bot User guide
+
+The source code for this bot is present in https://github.com/kubernetes-sigs/slack-infra/tree/main/slack-post-message and it is deployed using https://github.com/kubernetes/k8s.io/tree/main/apps/slack-infra
+
+## Prerequisites
+
+### Permissions
+
+Reach out to the 'sig-contribex-comms' channel in kubernetes slack for permissions or for posting something on your behalf.
+
+### Adding bot to a channel
+
+In order for the bot to send message to a channel, it needs to be part of the channel first and before you add it, you need to join the channel. Although we have it added in most of the channels, it still might be required to be added to a new channel. Follow the below steps for doing this
+
+
+1. Go to the 'More' option in the top left side and then go to 'Apps'
+![](https://i.imgur.com/Dk26HwP.png)
```

Comment:
we can have the assets within a folder in our GH repo, rather relying on 3rd parties.

Reply:
Yes, please put it all in our repos.

Reply:
added in folders. PTAL,